recipes/mirnature/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set version = "1.0" %}
+{% set sha256 = "7727f8c3e2f55e7ce74707e5c24d256d065dffc5c7704b0e06e04f685b830746" %}
+
+package:
+  name: mirnature
+  version: '{{version}}'
+
+source:
+  url: https://github.com/Bierinformatik/miRNAture/archive/v{{ version }}.tar.gz
+  sha256: '{{sha256}}'
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  host:
+  - perl=5.26.2
+  - perl-module-build=0.4224
+  - perl-getopt-long=2.50
+  - perl-sub-name=0.21
+  - perl-yaml-tiny=1.73  
+  - perl=5.26.2
+  - perl-bioperl=1.7.2
+  - perl-getopt-long=2.50
+  - perl-module-build=0.4224
+  - perl-moose=2.2011
+  - perl-moosex-types=0.50
+  - perl-moosex-types-path-class=0.09
+  - perl-regexp-common=2017060201
+  - perl-file-copy-recursive=0.45
+  - perl-file-share=0.25
+  - perl-file-sharedir=1.116
+  - perl-statistics-r=0.34=pl526r351_3
+  - r-base=3.5.1
+  - r-dplyr=0.8.5
+  - viennarna=2.4.17
+  
+  run:
+  - blast=2.9.0
```

Comment:
You have a lot of pins to an exact version. I would recommend relaxinf the version constraints, and
allow newer versions. Specifically for blast you should consider using 2.10.1 if possible

Reply:
Blast 2.9 is required as it is the tested version, 2.10 may change output, for this release of the software only 2.9 tests are available